lms/static/js/learner_dashboard/views/program_details_view.js

```diff
@@ -11,6 +11,7 @@ import HeaderView from './program_header_view';
 import SidebarView from './program_details_sidebar_view';
 
 import pageTpl from '../../../templates/learner_dashboard/program_details_view.underscore';
+import updatedPageTpl from '../../../templates/learner_dashboard/program_details_tab_view.underscore';
```

Comment:
=>`updatedPageTpl` please update it to be more readable e.g. that will convey that it's a tab view page.